# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/colo/relocation_fix-1.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Submitted By: Joe Ciccone <jciccone@linuxfromscratch.org>
Date: 2006-06-11
Initial Package Version: 1.21
Upstream Status: Unknown
Origin: Joe Ciccone
Description: Fixes relocation errors when linking.

diff -Naur colo-1.21.orig/stage1/include/lib.h colo-1.21/stage1/include/lib.h
--- colo-1.21.orig/stage1/include/lib.h	2006-01-17 18:03:58.000000000 -0500
+++ colo-1.21/stage1/include/lib.h	2006-07-11 01:01:29.000000000 -0400
@@ -47,8 +47,8 @@
 
 /* fast.c */
 
-extern void * __attribute__((section(".data"))) _memcpy_w(void *, const void *, size_t);
-extern void * __attribute__((section(".data"))) _memset_w(void *, int, size_t);
+extern void * _memcpy_w(void *, const void *, size_t);
+extern void * _memset_w(void *, int, size_t);
 
 #endif
 
